.changeset/odd-maps-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/underscore-redirects': minor
+---
+
+Adds the base path in front for input paths
```

Comment:
```suggestion
Adds the base path as prefix for input paths
```
I like the way you said it in the PR description better.  😄  I think "prefix" holds more meaning in context.